packages/babel-parser/src/parser/expression.js

```diff
@@ -2611,4 +2628,24 @@ export default class ExpressionParser extends LValParser {
 
     return ret;
   }
+
+  // https://github.com/tc39/proposal-js-module-blocks
+  parseModuleExpression(): N.ModuleExpression {
+    this.expectPlugin("jsModuleBlocks");
```

Comment:
I personally prefer `moduleBlocks` for brevity.

Reply:
5dd532b :+1: